variables.tf

```diff
@@ -19,6 +19,12 @@ variable "region" {
   description = "The IBM Cloud region where the cluster will be provisioned."
 }
 
+variable "use_private_endpoint" {
+  type        = bool
+  description = "Enable if the Terraform execution environment requires access to private endpoints."
```

Comment:
Can we come up with a better description here? Like what exactly will happen if this is set to true? Something like:

> Set this to true to force all api calls to use the IBM Cloud private endpoints